packages/dart_frog_cli/pubspec.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
   cli_completion: ^0.5.0
   dart_frog_gen: ^2.0.0
   equatable: ^2.0.5
-  mason: 0.1.0-dev.57
+  mason: 0.1.0-dev.58
```

Comment:
We require to bump the Dart environment to ^3.5.0 since mason requires precisely that.